pkg/sql/pg_catalog.go

```diff
@@ -1153,6 +1153,31 @@ https://www.postgresql.org/docs/9.5/catalog-pg-depend.html`,
 					depTypeAuto,          // deptype
 				)
 			}
+
+			reportViewDependency := func(dep *descpb.TableDescriptor_Reference) error {
+				for _, colID := range dep.ColumnIDs {
+					if err := addRow(
+						pgClassTableOid,                //classid
```

Comment:
can you also add a comment here that described how we don't use pg_rewrite (and link to issue https://github.com/cockroachdb/cockroach/issues/57417)

Reply:
Done